src/SelectField/SelectField.js

```diff
@@ -180,6 +185,7 @@ class SelectField extends Component {
       <TextField
         {...other}
         style={style}
+        defaultText={hintText || floatingLabelText ? '' : defaultText}
```

Comment:
I'm asking because `defaultText` doesn't seems to have any impact on the `TextField` component. And React is throwing a warning about leaking property.